rust/ql/src/queries/security/CWE-770/UncontrolledAllocationSize.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name Uncontrolled allocation size
+ * @description Allocating memory with a size controlled by an external user can result in
```

Comment:
The description LGTM but perhaps (feel free to ignore) we should mention why it is bad to allocate arbitrary amounts of memory (DoS)